boltstub/__init__.py

```diff
@@ -154,14 +154,21 @@ def finish(self):
 
     def start(self):
         if self.script.context.restarting or self.script.context.concurrent:
-            self.server.serve_forever()
+            self._serve_forever()
         else:
             self.server.handle_request()
             self.server.server_close()
 
     def _close_socket(self):
         self.server.socket.close()
 
+    def _serve_forever(self):
+        try:
+            self.server.serve_forever()
+        except OSError:
+            msg = "caught OS Error in stub._server_forever"
+            print(msg)
```

Comment:
a) you could inline the string :shrug: 
b) more importantly: why do we want to silently ignore OS errors?